rs-react/src/components/Card/CardTrait.tsx

```diff
@@ -11,28 +10,22 @@ type Props = {
   value: Gender | Species;
 };
 
-export class CardTrait extends Component<Props> {
-  getClass(value: string, type: string) {
-    if (type === 'gender') {
-      if (value === 'Male') return 'male';
-      if (value === 'Female') return 'female';
-    }
-
-    if (type === 'species') {
-      if (value === 'Human') return 'human';
-      if (value === 'Alien') return 'alien';
-    }
+function getClass(value: string, type: string) {
```

Comment:
```suggestion
return type === 'a' || type === 'b' ? value.toLowerCase() : ''
```
But in general, its very bad design when the presentation depends on the implementation of an external API, it should depend on abstractions.

Reply:
Done